### PR TITLE
How to install OpenSergo Control Plane

### PR DESCRIPTION
**Pull request by panxiaojun233**

### CODE REVIEW

**Review by sczyh30 — APPROVED** [review]
LGTM

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/quick-start/opensergo-control-plane.md [hunk 1]
```diff
@@ -12,12 +12,34 @@ sidebar_position: 1
...
 
-TODO: helm chart
+```shell
+# 创建 opensergo-system 命名空间
+kubectl apply -f opensergo-control-plane/k8s/namespace.yaml
+
+# opensergo 控制面相关权限配置
+kubectl apply -f opensergo-control-plane/k8s/rbac/role.yaml
+kubectl apply -f opensergo-control-plane/k8s/rbac/rolebinding.yaml
+kubectl apply -f opensergo-control-plane/k8s/rbac/serviceaccounts.yaml
```
Comment (jnan806) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/quick-start/opensergo-control-plane.md:22]: need update to match new yaml file 
image

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/quick-start/opensergo-control-plane.md [hunk 2]
```diff
@@ -12,12 +12,34 @@ sidebar_position: 1
...
+kubectl apply -f opensergo-control-plane/k8s/crd/bases/fault-tolerance.opensergo.io_faulttolerancerules.yaml
+kubectl apply -f opensergo-control-plane/k8s/crd/bases/fault-tolerance.opensergo.io_ratelimitstrategies.yaml
+kubectl apply -f opensergo-control-plane/k8s/crd/bases/fault-tolerance.opensergo.io_throttlingstrategies.yaml
+kubectl apply -f opensergo-control-plane/k8s/crd/bases/traffic.opensergo.io_trafficerouters.yaml
+
+# 安装 opensergo 控制平面 deployment
+kubectl apply -f opensergo-control-plane/k8s/deployment.yaml
+
+# 创建 opensergo 控制平面 service
+kubectl apply -f opensergo-control-plane/k8s/service.yaml
```
Comment (jnan806) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/quick-start/opensergo-control-plane.md:36]: need update to match new yaml file 
image

**Review by jnan806 — COMMENTED** [review]
need another PR to update new yaml files

**Pull request merged**